ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh

```diff
@@ -76,27 +76,27 @@ function disable_default_catalogsource () {
 }
 
 # Create the ICSP for optional operators dynamiclly, but we don't use it here
-function create_icsp_by_olm () {
-    mirror_auths="${SHARED_DIR}/mirror_auths"
-    # Don't mirror OLM operators images, but create the ICSP for them.
-    echo "===>>> create ICSP for OLM operators"
-    run_command "oc adm catalog mirror -a ${mirror_auths} quay.io/openshift-qe-optional-operators/ocp4-index:latest ${MIRROR_REGISTRY_HOST} --manifests-only --to-manifests=/tmp/olm_mirror"; ret=$?
-    if [[ $ret -eq 0 ]]; then
-        run_command "cat /tmp/olm_mirror/imageContentSourcePolicy.yaml"
-        run_command "oc create -f /tmp/olm_mirror/imageContentSourcePolicy.yaml"; ret=$?
-        if [[ $ret -eq 0 ]]; then
-            echo "create the ICSP resource successfully"
-        else
-            echo "!!! fail to create the ICSP resource"
-            return 1
-        fi
-    else
-        echo "!!! fail to generate the ICSP for OLM operators"
-        # cat ${mirror_auths}
-        return 1
-    fi
-    rm -rf /tmp/olm_mirror 
-}
+# function create_icsp_by_olm () {
```

Comment:
What is the replacement of this ICSP?

Reply:
@anpingli I used `create_settled_icsp` instead of it.